scripts/modules/ai-services-unified.js

```diff
@@ -86,6 +87,57 @@ function _getCostForModel(providerName, modelId) {
 	};
 }
 
+// --- Provider Function Map ---
+// Maps provider names (lowercase) to their respective service functions
+const PROVIDER_FUNCTIONS = {
```

Comment:
We completely changed this, please merge from next and see the new implementation of providers